### PR TITLE
chore(roll): roll Playwright to 1.38.0

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes https://github.com/microsoft/playwright-python/issues/2061

### CODE REVIEW

File: playwright/_impl/_js_handle.py [hunk 1]
```diff
@@ -195,6 +197,12 @@ def parse_value(value: Any, refs: Optional[Dict[int, Any]] = None) -> Any:
         if "bi" in value:
             return int(value["bi"])
 
+        if "m" in value:
+            return dict(parse_value(value["m"], refs))
```
Comment (yury-s) [inline playwright/_impl/_js_handle.py:201]: Let's not support Map/Set in non-js ports ?

**Review by yury-s — APPROVED** [review]

**Review by yury-s — APPROVED** [review]

File: tests/async/test_evaluate.py [hunk 2]
```diff
@@ -69,6 +69,14 @@ async def test_evaluate_transfer_bigint(page: Page) -> None:
     assert await page.evaluate("a => a", 17) == 17
 
 
+async def test_should_transfer_maps(page):
+    assert await page.evaluate("() => new Map([[1, { test: 42n }]])") == {}
```
Comment (YusukeIwaki) [inline tests/async/test_evaluate.py:73]: @mxschmitt Hi, does this mean playwright-python doesn't support transfering Map or Set actually?

```
Map(1) { 1 => { test: 42n } }
Set(2) { 1, { test: 42n } }
```

Comment (mxschmitt) [reply tests/async/test_evaluate.py:73]: Hi yes, we decided to not implement Map/Set in the language ports for now. (Then last 2 years nobody asked for it)

Comment (YusukeIwaki) [reply tests/async/test_evaluate.py:73]: Thank you for sharing about the decision :)

**Pull request merged**